base_search_improved/README.rst

```diff
@@ -0,0 +1,93 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============
+Improved Search
+===============
+
+This module extends the search functionality to add simple shortcuts for
+more complex search conditions.
+
+Features:
+* Fuzzy search: use wildcards ("%") in place of spaces
```

Comment:
But it makes it more unpredictable. You lose control over your search (and it also has a performance penalty). It should be at least optional.